src/ucs/config/global_opts.h

```diff
@@ -105,7 +105,7 @@ typedef struct {
     size_t                     memtrack_limit;
 
     /* Profiling mode */
-    unsigned                   profile_mode;
+    uint64_t                   profile_mode;
```

Comment:
also `handle_errors` in this struct

Reply:
thanks, somehow I had changed it but lost some in the meantime.